GPU/Common/GPUStateUtils.cpp

```diff
@@ -959,7 +959,7 @@ void ApplyStencilReplaceAndLogicOpIgnoreBlend(ReplaceAlphaType replaceAlphaWithS
 bool IsColorWriteMaskComplex(bool allowFramebufferRead) {
 	// Restrict to Outrun temporarily (by uglily reusing the ReinterpretFramebuffers flag)
```

Comment:
Should we need change this commit ?

Reply:
Yeah, I should have removed that comment.